dot/parachain/bitfield-distribution/bitfield_distribution.go

```diff
@@ -235,19 +249,79 @@ func (b *BitfieldDistribution) processPeerConnectedEvent(event networkbridgeeven
 }
 
 func (b *BitfieldDistribution) processPeerDisconnectedEvent(event networkbridgeevents.PeerDisconnected) {
+	logger.Infof("peer disconnected: %s", event.PeerID)
+
 	b.mu.Lock()
 	delete(b.peerViews, event.PeerID)
 	b.mu.Unlock()
 }
 
 func (b *BitfieldDistribution) processNewGossipTopologyEvent(event networkbridgeevents.NewGossipTopology) error {
-	//TODO implement in #4357
-	panic("implement me")
+	logger.Infof("process NewGossipTopology event")
```

Comment:
this log looks like a trace than an info log, what about changing it?
```suggestion
	logger.Tracef("processing NewGossipTopology event")
```